docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
+  - Control implementation should be systematic and directly aligned to the internal and stakeholder requirements.
   - Mandatory
 ```
 
-## Quality management
+**Resource Allocation:** The process of assigning, distributing, and managing resources (such as personnel, finances, equipment, or time) within the TRE organisation to meet objectives and priorities effectively.
 
-_The ability of a TRE operator to measure and control quality of processes, documentation and outputs._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-resource-allocation
 
-### Document management
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure there are adequate resources to provide assurance and meet information governance requirements.
+  - There should be adequate access to funding to implement and maintain the TRE which is under the direct control of the TRE organisation.
+  - Mandatory
+```
+
+## Quality Management
+
+What the organisation does to measure and control quality of processes, documentation and outputs.
+
+**Document and SOP Management:** Creating, organising, updating, and controlling documents and Standard Operating Procedures (SOPs) within the TRE organisation.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-document-management
+:name: tab-ig-quality-management
 
 * - Statement
   - Guidance
   - Importance
-* - You must control all of your policies and standard operating procedures.
-  - This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE operators.
+* - You must ensure all policies & standard operating procedures relevant to the TRE organisation are controlled.
+  - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.
   - Mandatory
-* - You should use codified change processes when altering your policies and standard operating procedures.
-  -
-  - Recommended
-* - You could use version control to track changes to their policies and processes.
+* - You must ensure all policies & standard operating procedures relevant to the TRE organisation are version controlled and have codified change processes.
   - Version control includes recording dates of changes, person responsible for carrying out changes, and summary of changes.
-  - Optional
+  - Mandatory
 ```
 
-### Issue management
+**Quality Management Reporting:** The generation and dissemination of reports or dashboards that provide insights and metrics on the performance and effectiveness of quality management processes and activities.
 
-_The ability of a TRE operator to track deviations from stated policies._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-quality-management-reporting
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must measure the performance of information governance within the TRE with regular reporting available to top management.
```

Comment:
Who are top management?

Reply:
Maybe "senior management"? What's the generic term for people who have ultimate responsibility?

Reply:
I think @machintim meant something like "the people with overall responsibility for the TRE organisation" where "TRE organisation" is defined in #160. Worth adding to the glossary and/or finding a better name?

Reply:
```suggestion
* - You must measure the performance of information governance within the TRE with regular reporting available to the TRE organisation management team.
```